default-templates/emails/invalid-username.js

```diff
@@ -0,0 +1,30 @@
+module.exports.render = render
+
+function render (data) {
+  return {
+    subject: `Invalid username for account ${data.accountUri}`,
+
+    /**
+     * Text version
+     */
+    text: `Hi,
+
+We're sorry to inform you that the username for account ${data.accountUri} is not allowed after changes to username policy.
```

Comment:
Maybe refer to `username policy` somehow? I'd be wondering, what exactly is in there if I would receive such an e-mail.

Reply:
I don't think this is necessary, but open to be convinced otherwise.